noggin/tests/unit/controller/test_password_reset.py

```diff
@@ -81,6 +81,41 @@ def test_password_changes_user(
     )
 
 
+@pytest.mark.vcr()
+def test_password_form_with_otp(client, logged_in_dummy_user, dummy_user_with_otp):
+    """Verify that the password change form shows OTP form elements
+       when a user has OTP enabled"""
+    result = client.get("/user/dummy/settings/password")
+
+    page = BeautifulSoup(result.data, "html.parser")
+
+    currentpasswordinput = page.select("#currentpasswordinput .form-text")
+    assert len(currentpasswordinput) == 1
+    assert "Just the password, don't add the OTP token if you have one" in currentpasswordinput[
+        0
+    ].get_text(
+        strip=True
+    )
```

Comment:
Well that's a bit verbose (because of black's reformatting), I would have written something like:
```python
currentpasswordinput = page.select_one("#currentpasswordinput .form-text")
assert currentpasswordinput is not None
expected = "Just the password, don't add the OTP token if you have one"
assert expected in currentpasswordinput.get_text(strip=True)
```
And black should leave that alone, I think (not tested).